index.markdown

```diff
@@ -119,10 +119,11 @@ unless you have the financial ability to not care. Rent in major cities like SF
 
 <section id="Jobs-Perks" />
 ### Perks
-Some companies have cool perks like food, bikes, transit passes, happy hour,
+Some companies have cool perks like catered lunches/dinners, bikes, transit passes, happy hour,
 snacks, game rooms, pool tables, ping pong, foosball, museum passes, etc.
 
-It's worth checking out.
+It's worth checking out. Catered lunches/dinners can drastically help reduce your daily spending,
```

Comment:
Can you remove the line catered from this line and the above line. Companies have different ways of giving us free lunches and dinners.